app/src/main/java/com/seafile/seadroid2/util/Utils.java

```diff
@@ -380,6 +380,13 @@ public static String pathJoin (String first, String... rest) {
         return result.toString();
     }
 
+    public static String pathJoin(String dir) {
```

Comment:
Please call the function addSlash instead, it is hard to guess what an unary pathJoin will do when reading the code.